README.md

```diff
@@ -1,5 +1,3 @@
-# Provide
```

Comment:
Btw, I removed this because it's common practice on github and (especially) pub.dartlang.org. There is already an `<h1>` on those sites.